### ISSUE TITLE
Spotify++ Songs dont download

### ISSUE THREAD

**Issue by snaksd** [JMccormick264/PPAppsIPARepo#141]
Everything works fine but i cant download any songs, when I klick on the download butten it turns green but the songs arent downloading

**Comment by potatoeux** [comment]
Yeah I'm guessing Spotify servers reject the request because you actually aren't a premium user.
Nothing the tweak can do here 🤷
